examples/widgets/cursor.py

```diff
@@ -20,7 +20,7 @@
 ax.set_xlim(-2, 2)
 ax.set_ylim(-2, 2)
 
-# set useblit = True on gtkagg for enhanced performance
+# Set useblit=True on some backends for enhanced performance.
```

Comment:
`"useblit=True" improves the performance on most backends`? (gtk3cairo (fundamentally) cannot support blitting and I'm not sure what the situation is with nbagg but all other major backends do support blitting.)